workflows/scrnaseq.nf

```diff
@@ -67,7 +71,8 @@ workflow SCRNASEQ {
     ch_salmon_index = params.salmon_index ? file(params.salmon_index) : []
 
     //star params
-    ch_star_index = params.star_index ? file(params.star_index) : []
+    star_index = params.star_index ? file(params.star_index, checkIfExists: true) : null
+    ch_star_index = star_index ? [[id: star_index.baseName], star_index] : []
```

Comment:
Is it correct to call this a `ch_` prefix when it's not actually in a channel object yet (it seems)? 

I'm assuming you're fine with the code, I'm just checking something hasn't been missed

Reply:
No, no standards. 

Just checking that a channel generation wasn't missing anywhere, where it should have been :)